### ISSUE TITLE
Fonts 1.0.0-beta16 with ImageSharp 2.1.3 crashes with NullReferenceException when measuring text

### ISSUE THREAD

**Issue by myblindy** [SixLabors/Fonts#292]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have verified that I am running the latest version of Fonts
- [X] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [X] I have searched [open](https://github.com/SixLabors/Fonts/issues) and [closed](https://github.com/SixLabors/Fonts/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

As in the title, upgrading to ImageSharp 2.* (and Fonts to 1.0.0-beta16 to match) now crashes during measuring.

### Steps to Reproduce

```
using SixLabors.Fonts;

var fontCollection = new FontCollection();
var regularFontFamily = fontCollection.Add(@"Font\OpenSans-Regular.ttf");
var font = regularFontFamily.CreateFont(25);

var fontRect = TextMeasurer.Measure("V", new(font));
```

Full exception:
```
System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=SixLabors.Fonts
  StackTrace:
   at SixLabors.Fonts.Tables.AdvancedTypographic.GPosTable.GetFeatureLookups(Tag& stageFeature, ScriptClass script) in /_/src/SixLabors.Fonts/Tables/AdvancedTypographic/GPosTable.cs:line 210
```

![image](https://user-images.githubusercontent.com/541998/180284106-974283e8-3905-4aa5-8a73-6eabc52f0d57.png)

Similar code works fine with ImageSharp 1.* and the corresponding Fonts (with a few method name changes, of course). 
You can get the full repository at https://github.com/myblindy/font_test.

### System Configuration

- Fonts version: 1.0.0-beta16
- Other Six Labors packages and versions: ImageSharp 2.1.3, Drawing 1.0.0-beta14
- Environment (Operating system, version and so on): Windows 10
- .NET Framework version: 6.0.4
- Additional information: N/A

**Comment by myblindy** [comment]
I just tried Fonts 1.0.0-beta17 with same results.

**Comment by JimBobSquarePants** [comment]
See #276 and #266. This is a duplicate. You’ll need the latest MyGet release:

**Comment by myblindy** [comment]
For what it's worth, it actually worked with the woff2 version of the font, so I'm using that for now.

**Comment by JimBobSquarePants** [comment]
Yep that's because the WOFF font will not include the redundant features. I've released 1.0.0-beta18 on NuGet now which contains the fix.
